packages/o365_metrics/data_stream/groups_activity_group_detail/manifest.yml

```
@@ -0,0 +1,88 @@
+title: Microsoft 365 Office365 Groups Activity Group Detail
+type: metrics
+ilm_policy: metrics-o365_metrics.groups_activity_group_detail-default_policy
+streams:
+  - input: cel
+    title: Microsoft 365 Office365 Groups Activity Group Detail
+    description: Collect Microsot Office365 Groups Activity Group Detail metrics via Microsoft Graph API using CEL Input.
+    template_path: cel.yml.hbs
+    enabled: false
+    vars:
+      - name: report_types
+        title: Report Types
+        description: Select the type of report to ingest data.
+        type: text
+        multi: true
+        required: true
+        show_user: true
+        default:
+          - 'Office365 Groups Activity Group Detail'
+      - name: interval
+        type: text
+        title: Interval
+        description: >-
+          Interval between two API calls. Supported units for this parameter are h/m/s.
```

Nit, same description as other data streams. Same for all manifest files.
```suggestion
          Interval of API polling. Interval supported in seconds, minutes and hours.
```

---

Updated in [767e86c](https://github.com/elastic/integrations/pull/12256/commits/767e86cd0f32b4905a04ba9283e84818781f53a1)